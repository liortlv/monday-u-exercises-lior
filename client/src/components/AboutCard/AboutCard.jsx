import aboutCardCSS from "./AboutCard.module.css";
import profilePic from "../../assets/profilepic.jpeg";

function AboutCard() {
   return (
      <div className={aboutCardCSS.card}>
         <h1 className={aboutCardCSS.title}>About</h1>
         <p className={aboutCardCSS.info}>
            This is an exercise, developed as part of the Monday-U Full Stack course.
         </p>
         <img src={profilePic} alt="profile" className={aboutCardCSS.profile} />
         <h2 className={aboutCardCSS.name}>Lior Tal</h2>
      </div>
   );
}

export default AboutCard;
