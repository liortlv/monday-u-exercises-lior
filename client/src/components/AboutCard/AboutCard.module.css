.card {
   display: flex;
   flex-direction: column;
   align-items: center;
   background-color: aliceblue;
   width: 59rem;
   min-width: 45rem;
   height: 69rem;
   border-radius: 8px;
}

.title {
   font-size: 4rem;
   padding: 1rem;
   margin-top: 2rem;
}

.info {
   font-size: 1.7rem;
   width: 50rem;
   margin-bottom: 2rem;
   text-align: left;
}

.profile {
   width: 30rem;
   height: auto;
   border: 1px solid black;
   border-radius: 6px;
}

.name {
   font-size: 3rem;
   margin-top: 1rem;
}
