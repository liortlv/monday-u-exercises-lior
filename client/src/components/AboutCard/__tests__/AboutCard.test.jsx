import renderer from "react-test-renderer";
import AboutCard from "../AboutCard";

it("renders correctly", () => {
   const tree = renderer.create(<AboutCard />).toJSON();
   expect(tree).toMatchSnapshot();
});
