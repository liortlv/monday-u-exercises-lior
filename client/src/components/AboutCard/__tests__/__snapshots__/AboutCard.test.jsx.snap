// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card"
>
  <h1
    className="title"
  >
    About
  </h1>
  <p
    className="info"
  >
    This is an exercise, developed as part of the Monday-U Full Stack course.
  </p>
  <img
    alt="profile"
    className="profile"
    src="profilepic.jpeg"
  />
  <h2
    className="name"
  >
    Lior Tal
  </h2>
</div>
`;
